src/storage/remote/storage.c

```diff
@@ -95,41 +66,47 @@ storageRemoteInfoParse(ProtocolClient *client, StorageInfo *info)
     FUNCTION_TEST_END();
 
     info->type = storageRemoteInfoParseType(strPtr(protocolClientReadLine(client))[0]);
-    info->userId = jsonToUInt(protocolClientReadLine(client));
-    info->user = jsonToStr(protocolClientReadLine(client));
-    info->groupId = jsonToUInt(protocolClientReadLine(client));
-    info->group = jsonToStr(protocolClientReadLine(client));
-    info->mode = jsonToUInt(protocolClientReadLine(client));
     info->timeModified = (time_t)jsonToUInt64(protocolClientReadLine(client));
 
     if (info->type == storageTypeFile)
         info->size = jsonToUInt64(protocolClientReadLine(client));
 
-    if (info->type == storageTypeLink)
-        info->linkDestination = jsonToStr(protocolClientReadLine(client));
+    if (info->level >= storageInfoLevelDetail)
+    {
+        info->userId = jsonToUInt(protocolClientReadLine(client));
+        info->user = jsonToStr(protocolClientReadLine(client));
+        info->groupId = jsonToUInt(protocolClientReadLine(client));
+        info->group = jsonToStr(protocolClientReadLine(client));
+        info->mode = jsonToUInt(protocolClientReadLine(client));
+
+        if (info->type == storageTypeLink)
+            info->linkDestination = jsonToStr(protocolClientReadLine(client));
```

Comment:
Same comment about storageTypeLink as above in src/storage/remote/protocol.c

Reply:
Same response as above... :)